### DISCUSSION TITLE
Install dev mode in virtualenv

### DISCUSSION THREAD

**Discussion by jaykumar16** [Project-MONAI/MONAI#6022] [Q&A] [answered]
I am having some trouble installing the package in the virtual environment with development mode. 

It only installs with sudo to the whole system. 

python setup.py develop

SetuptoolsDeprecationWarning: setup.py install is deprecated. Use build and pip and other standards-based tools.
  warnings.warn(
running egg_info
error: Cannot update time stamp of directory 'monai.egg-info'

Please help.

**Comment by wyli** [comment, marked as answer]
I came across the same issue earlier, the cause was that `monai.egg-info` was created by root within a docker. `sudo rm -rf monai.egg-info` then `python setup.py develop --user` worked for me.

**Reply by jaykumar16** [reply]
Thank you so much. It worked.
